x-pack/test/security_solution_cypress/cypress/urls/navigation.ts

```diff
@@ -25,8 +25,11 @@ export const CSP_DASHBOARD_URL = '/app/security/cloud_security_posture/dashboard
 export const CSP_FINDINGS_URL = '/app/security/cloud_security_posture/findings/vulnerabilities';
 
 export const RULES_LANDING_URL = '/app/security/rules/landing';
+export const RULES_MANAGEMENT_URL = '/app/security/rules/management';
```

Comment:
There is already [RULES_MANAGEMENT_URL constant](https://github.com/elastic/kibana/blob/main/x-pack/test/security_solution_cypress/cypress/urls/rules_management.ts#L8) defined.